qcfractal/storage_sockets/sql_models.py

```diff
@@ -810,14 +810,22 @@ class ServiceQueueORM(Base):
     procedure_id = Column(Integer, ForeignKey("base_result.id"), unique=True)
     procedure_obj = relationship(BaseResultORM, lazy='joined')
 
+    priority = Column(Integer, default=int(PriorityEnum.NORMAL))
+    created_on = Column(DateTime, default=datetime.datetime.utcnow)
```

Comment:
We need to write a DB migration to be able to add this, and in indices to the existing production DB.
Please don't merge this pull request until we get the migration script as this will break the production if pushed by mistake.

Reply:
Isn't this expected that between versions we will continuously make database changes? The primary fractal server will always be updated to a release version.